packages/TorneloScoresheet/src/types/ChessMove.ts

```diff
@@ -24,21 +34,29 @@ export enum PlyTypes {
   MovePly,
   SkipPly,
 }
+
+// Information about a given ply
 export type PlyInfo = {
   moveNo: number;
   startingFen: string;
   player: PlayerColour;
   type: PlyTypes;
 };
 
+// A move ply is a recorded ply that involved
+// a piece moving across the board
 export type MovePly = {
   move: MoveSquares;
   type: PlyTypes.MovePly;
   promotion?: PieceType;
 } & PlyInfo;
 
+// A skip ply is a recorded ply that has been
+// marked as "skipped" - i.e. this is a placeholder
+// for a ply will be filled in later
 export type SkipPly = {
```

Comment:
Not entirely sure what this does maybe I'm not understanding it. Is this in place for moves where the turn of white/black is skipped in arbiter mode?

Reply:
Sort of! It is a placeholder for a move that has been skipped by the player and will be filled in later. It will be helpful for when they are under time pressure and don't have time to fill in the moves

Reply:
Ah sweet, I see now. Cheers